engine/BUILD.bazel

```diff
@@ -0,0 +1,66 @@
+load("//:version.bzl", "PROJECT_VERSION")
```

Comment:
ditto about *.bzl. (again just optics not really necessary to do any actions here)

Reply:
I want the place where version of the whole project is defined to be at repository root. It needs to be changed often, it's important, and it should be obvious. I don't want it languishing in `tools/bazel/version.bzl` where one would expect... helper tools, not important project configuration.

Perhaps I should rename it to `config.bzl` or something.

For the other one, `plugin.bzl` I guess that could go somewhere else.